test/celeritas/CMakeLists.txt

```diff
@@ -163,6 +163,9 @@ celeritas_add_test(ext/GeantImporter.test.cc
 celeritas_add_test(ext/GeantVolumeMapper.test.cc ${_needs_geant4}
   LINK_LIBRARIES ${Geant4_LIBRARIES} Celeritas::orange)
 celeritas_add_test(ext/RootImporter.test.cc ${_needs_root})
+celeritas_add_test(ext/RootJsonDumper.test.cc ${_needs_root}
+  LINK_LIBRARIES nlohmann_json::nlohmann_json
```

Comment:
This may be a silly question, but why do we need to link nlohmann here? ROOT classes are already linked against it, no?

Reply:
Oh I intended to modify the JSON export directly but ended up changing my mind. You're right that it's not needed now.